litebird_sim/destriper/__init__.py

```diff
@@ -216,11 +217,15 @@ def __init__(
         )
 
     def local_intervals(self, _):
+        start_time = (
```

Comment:
Hi @ziotom78, I added this small piece here to handle obs.start_time passed as an astropy.time.Time object. In this case, `obs.start_time` is not a number and we have to call `obs.start_time.cxcsec`